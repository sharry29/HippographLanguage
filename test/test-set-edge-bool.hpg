int main() {
	graph<bool:bool, string> g = [true:true; false: false];
	g.set_edge(true; false); (* set weight to empty *)
  g.print();
  return 0;
}
