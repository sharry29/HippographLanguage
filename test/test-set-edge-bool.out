true:true -> [false:false (null)]
false:false -> []
